scripts/kg_build.py

```
@@ -0,0 +1,48 @@
+    args = parser.parse_args()
+
+    ensure_env("OPENAI_API_KEY")
+    ensure_env("NEO4J_URI")
+    ensure_env("QDRANT_URL")
+
+    log = {
+        "timestamp": _dt.datetime.utcnow().isoformat() + "Z",
+        "operation": "kg_build",
+        "source": args.source,
+        "status": "skipped",
+        "message": "Stub implementation – full pipeline delivered in Story 2.5",
```

Ruff (0.13.1)

String contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF001)